Library/Formula/keybase.rb

```diff
@@ -1,33 +1,32 @@
 class Keybase < Formula
   desc "Command-line interface to Keybase.io"
   homepage "https://keybase.io/"
-  url "https://github.com/keybase/node-client/archive/v0.8.23.tar.gz"
```

Comment:
The old client (`node-client`) is still referenced on the download page, Is that being updated? Just don't want to confuse people who expect one thing from the website and end up with another from us, ideally.